app/jinja_filters.py

```diff
@@ -134,7 +152,7 @@ def get_format_date(value):
 
 @pass_eval_context  # type: ignore
 @blueprint.app_template_filter()
-def format_datetime(context, date_time):
+def format_datetime(context: nodes.EvalContext, date_time: str) -> str:
```

Comment:
Worth double checking but I think `date_time` parameter needs to be `datetime` type.